apps/omg_performance/lib/omg_performance/http_rpc/watcher_client.ex

```diff
@@ -14,72 +14,167 @@
 
 defmodule OMG.Performance.HttpRPC.WatcherClient do
```

Comment:
too bad this is implemented this way. you get a client implementation in elixir for free if you'd use swagger API definition!

Reply:
yyyyyeahh, considering this already exists, would you say it's better to try to swap out for the swagger client or meh? How much effort do you think it'd take?

Reply:
It's not too difficult (look at `priv/`) but I'm not sure how to get this into omg_perfromance...

Reply:
OK, since this got looped into this repo under `priv` it seems like the path is much more paved now. Will `priv` be visible and accessible from `omg_performance` level though?

Another question is whether we want to plunge into this now kinda "separately", or maybe when those tests get rephrased into the `specs` language (or whatever approach is used for the integration tests).